ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -2107,6 +2108,36 @@ impl<'a> Assemble<'a> {
         }
     }
 
+    fn cg_bitcast(&mut self, iidx: InstIdx, inst: &jit_ir::BitCastInst) {
+        let src_type = self.m.type_(inst.val(self.m).tyidx(self.m));
+        let dest_type = self.m.type_(inst.dest_tyidx());
+
+        match (src_type, dest_type) {
+            (jit_ir::Ty::Float(_), jit_ir::Ty::Float(_)) => {
+                todo!();
+            }
+            (jit_ir::Ty::Float(_), _gp_ty) => {
+                todo!();
+            }
+            (gp_ty, jit_ir::Ty::Float(_)) => {
+                let [src_reg] = self.ra.assign_gp_regs(
+                    &mut self.asm,
+                    iidx,
+                    [RegConstraint::Input(inst.val(self.m))],
+                );
+                let [tgt_reg] =
+                    self.ra
+                        .assign_fp_regs(&mut self.asm, iidx, [RegConstraint::Output]);
+                match gp_ty.byte_size() {
```

Comment:
I think the `None` case here means there's invalid IR? If so, we should `unwrap`.

Reply:
Fixed in 417229a6da535b945216c1895c27a3ddc9059cbe